### DISCUSSION TITLE
Mailing list dead?

### DISCUSSION THREAD

**Discussion by reagle** [jgm/pandoc#9344] [Q&A] [answered]
pandoc-discuss was getting hammered with spam, and I posted a question about it, and that's the last email I've received. I just went to check Google Groups and the list isn't available: "Banned content warning".

<img width="866" alt="SCR-20240117-otyd" src="https://github.com/jgm/pandoc/assets/596814/b573a25b-929f-4672-9f9a-589f66150ec7">

**Comment by jgm** [comment, marked as answer]
See #9250

**Comment by jgm** [comment]
Searchable archive of pandoc-discuss: https://inbox.vuxu.org/pandoc-discuss/
